module/Application/view/application/video/index.phtml

```diff
@@ -8,10 +8,10 @@
 	<br clear="left">
 
     <div class="grid" style="padding-left: 10px; margin-top: 20px;">
-        <div class="videoContent">
-            <video id="agPlayer" controls preload="none" width="960" height="540" style="width: 100%; height: 100%;">
-                <source src="http://www.youtube.com/watch?v=ZBSX1AB6LcM" type="video/youtube">
-            </video>
+        <div class="media-wrapper">
+            <iframe width="960" height="540"
+                    src="https://www.youtube.com/embed/ZBSX1AB6LcM"
+                    frameborder="0" allowfullscreen></iframe>
```

Comment:
I don't understand these changes; why are you reverting to an iframe instead of using the HTML5 video tag?

Reply:
With video tag it's not gonna work on all browsers. Then we have to use some JS libraries to add support - previously we used.... hm.. mediaelement (?) Now 'responsibility' to display the video properly we moved on YouTube and I think it's better, as we are using videos only from YT.